src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -1477,6 +1477,52 @@ .maxstack  2
 ");
         }
 
+        [Fact, WorkItem(50170, "https://github.com/dotnet/roslyn/issues/50170")]
+        public void StaticCtor()
+        {
+            var src = @"
+record R(int x)
+{
+    static R() { }
+}
+";
+
+            var comp = CreateCompilation(src);
+            comp.VerifyEmitDiagnostics();
```

Comment:
>comp.VerifyEmitDiagnostics(); [](start = 12, length = 29)

Consider running the assembly and observing that the constructor is invoked as expected.